builders/pallets-precompiles/precompiles/randomness.md

```diff
@@ -152,128 +152,161 @@ The happy path for a randomness request is shown in the following diagram:
+2. Make sure **Injected Provider - Metamask** is selected in the **ENVIRONMENT** dropdown. Once you select **Injected Provider - Metamask**, you might be prompted by MetaMask to connect your account to Remix
 3. Make sure the correct account is displayed under **ACCOUNT**
-4. You'll need to pay the required deposit for a randomness request. For this contract, the required deposit is {{ networks.moonbase.randomness.req_deposit_amount.dev }} DEV. So set the value to `{{ networks.moonbase.randomness.req_deposit_amount.wei }}` and choose **Wei** from the dropdown on the right
-5. Ensure **RandomnessLotteryDemo - RandomnessLotteryDemo.sol** is selected in the **CONTRACT** dropdown
-6. Next to **Deploy** enter the source of randomness. This corresponds to the [`RandomnessSource`](#enums) enum. For local VRF, enter `0`, and for BABE epoch randomness, enter `1`. To follow along with this example, you can select `0` and click **Deploy**
+4. Enter the deposit amount in the **VALUE** field, which is `{{ networks.moonbase.randomness.req_deposit_amount.wei }}` in Wei (`{{ networks.moonbase.randomness.req_deposit_amount.dev }}` Ether)
+5. Ensure **RandomNumber - RandomNumber.sol** is selected in the **CONTRACT** dropdown
+6. Click **Deploy**
 7. Confirm the MetaMask transaction that appears by clicking **Confirm**
 
-![Deploy RandomnessLotteryDemo](/images/builders/pallets-precompiles/precompiles/randomness/randomness-3.png)
-
-The **RANDOMNESSLOTTERYDEMO** contract will appear in the list of **Deployed Contracts**.
+![Deploy the random number generator contract in Remix.](/images/builders/pallets-precompiles/precompiles/randomness/new/randomness-4.png)
 
-### Participate in the Lottery {: #participate-in-lottery }
+The **RANDOMNUMBER** contract will appear in the list of **Deployed Contracts**.
 
-The default contract has a minimum requirement of three participants. To participate you can take the following steps:
+### Submit a Request to Generate a Random Number {: #request-randomness }
 
-1. Make sure you've switched to the account you want to participate with in MetaMask. You can verify the account that is connected under **ACCOUNT** 
-2. Enter the amount you want to contribute to the lottery in the **VALUE** field. It must be greater than or equal to the `PARTICIPATION_FEE` which is set to `100000 gwei` in the default contract
-3. Click on **participate**
-4. Confirm the transaction in MetaMask
+To request randomness, you're going to use the `requestRandomness` function of the contract, which will require you to submit a deposit as defined in the Randomness Precompile. You can submit the randomness request and pay the deposit by taking these steps:
 
-![Participate in the lottery](/images/builders/pallets-precompiles/precompiles/randomness/randomness-4.png)
+1. Enter an amount in the **VALUE** field for the fulfillment fee, it must be equal to or greater than the minimum fee specified in the `RandomNumber` contract, which is `500000` Gwei
```

Comment:
Is `500000` a variable that we need to put in the variables file?

Reply:
no, this is just for this example but users can test and configure this number

Reply:
added a comment to the contract where the fulfillment fee is defined